qtcreator/cmut__qtcreator__import_qml.cmake

```diff
@@ -0,0 +1,58 @@
+# In order to use qtcreator inspector to load modules QML dynamically
+# and propose code completion in multi libraries environment we have to export
+# QML files and set QML_IMPORT_PATH variable.
+
+function(cmut__qtcreator__import_qml target)
+
+    get_target_property(_target_dependencies ${target} LINK_LIBRARIES)
+
+    __cmut__qtcreator__get_qml_import_paths(_qml_directories "${_target_dependencies}")
+
+    set( QML_IMPORT_PATH "${_qml_directories}" CACHE STRING "Qml import path" FORCE)
+
+endfunction()
+
+
+function(__cmut__qtcreator__get_dependency_install_directory dependency_install_directory dependency)
+# PREDICATE Library directory is one delph children of install directory
+
+    if( ${CMAKE_BUILD_TYPE} STREQUAL "Release" )
+         get_target_property(_dependency_location ${dependency} IMPORTED_LOCATION_RELEASE)
```

Comment:
try the current config, then try  the default config
```
string(TOUPPER ${CMAKE_BUILD_TYPE} BUILD_TYPE_UPPER)
get_target_property(_dependency_location ${dependency} IMPORTED_LOCATION_${CMAKE_BUILD_TYPE})
if(NOT _dependency_location)
    get_target_property(_dependency_location ${dependency} IMPORTED_LOCATION)
endif()
```

Reply:
Done